SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -7,6 +7,79 @@
 
 #include "SpatialGDKEditorSettings.generated.h"
 
+USTRUCT()
+struct FWorldLaunchSection
+{
+	GENERATED_BODY()
+
+	FWorldLaunchSection()
+		: Dimensions(2000, 2000)
+		, ChunkEdgeLenghtMeters(50)
+		, StreamingQueryInterval(4)
+		, SnapshotWritePeriodSeconds(0)
+	{
+		LegacyFlags.Add(TEXT("streaming_query_diff"), TEXT("true"));
```

Comment:
Looks like this flag is no longer supported.

Reply:
Yep will remove it now